Userland/Libraries/LibVideo/VP9/MotionVector.h

```diff
@@ -10,24 +11,60 @@
 
 namespace Video::VP9 {
 
-class MotionVector {
+struct MotionVector {
 public:
-    MotionVector() = default;
-    MotionVector(MotionVector const& other);
-    MotionVector(u32 row, u32 col);
+    constexpr MotionVector() = default;
+    constexpr MotionVector(i32 row, i32 col)
+        : m_row(row)
+        , m_column(col)
+    {
+    }
+    constexpr MotionVector(MotionVector const& other)
```

Comment:
in case the commits get lost/shuffled, MotionVector (old MV) should have = default'd special member functions.